docs/src/Table.doc.js

```diff
@@ -259,6 +259,362 @@ card(
   />,
 );
 
+card(
+  <Example
+    id="stickyColumn"
+    name="Example: Sticky Column"
+    description="Try scrolling horizontally to see the first column remain in place."
+    defaultCode={`
+<Box width={"50%"} overflow="auto">
+<Table maxHeight={200} stickyColumns={1}>
```

Comment:
These code examples are pretty hefty; can you break things up with some whitespace for readability? Since they start collapsed, I'm not really concerned with taking up more real estate